### PR TITLE
21.3w fb water test merge

### PR DESCRIPTION
**Pull request by dnicolalde**
#### Rationale
The water system is working correctly and passing test on TeamCity, time to merge into staging.

#### Related Pull Requests
* 

#### Changes
Catching up with staging branch.

### CODE REVIEW

File: WNPRC_EHR/resources/queries/study/chairing.js [hunk 1]
```diff
@@ -88,3 +88,49 @@ function onUpsert(helper, scriptErrors, row, oldRow){
...
 
 }
+function setDescription(row,helper){
+    var description = new Array();
+
+    if (row.description){
+        description.push(EHR.Server.Utils.nullToString(row.description));
+    }
+    if (row.location){
+        console.log(JSON.stringify(row.location));
```
Comment (gottfred) [inline WNPRC_EHR/resources/queries/study/chairing.js]: not sure if intentional, but there is some logging left in here

File: WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/husbandry/WaterCalendar.jsp [hunk 2]
```diff
@@ -1155,7 +1150,7 @@
                             window.open(newWaterOrder,'_blank');
 
                         }
-                        if (!response.success){
+                        else if (!response.success){
```
Comment (gottfred) [inline WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/husbandry/WaterCalendar.jsp]: this could just be an else

**Review by gottfred — APPROVED** [review]
I left a couple of comments, but nothing that should cause any issues. Otherwise looks good

File: WNPRC_EHR/resources/queries/study/chairing.js (hunk 1 above)
Comment (dnicolalde) [reply WNPRC_EHR/resources/queries/study/chairing.js]: Yes this was overlooked

File: WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/husbandry/WaterCalendar.jsp (hunk 2 above)
Comment (dnicolalde) [reply WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/husbandry/WaterCalendar.jsp]: it  was giving me some problems when it was just an if not sure why

**Comment by dnicolalde** [comment]
Additional changes to fix a problem with the frequency drop down.

**Pull request merged**